codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -296,6 +296,7 @@ final DefaultPropertyKey verifyKey(PropertyKey key) {
         private int totalChildWeights;
         private int prioritizableForTree = 1;
         private boolean resetSent;
+        private boolean headerSent;
```

Comment:
Right now the flow controller doesn't know anything about the frames being passed in. I don't want to make such a large change in this PR.

It's clear that this PR is a stop-gap until we decide how we want to properly handle this, so I'm kicking the can a bit here.